db/seeds.rb

```diff
@@ -77,7 +82,8 @@
   reason_to_donate: "I support women's reproductive freedom",
   slug: 'center-for-reproductive-rights',
   info_url: 'http://www.reproductiverights.org/',
-  donation_url: 'https://secure3.convio.net/cfrr/site/Donation2?df_id=4620&4620.donation=form1&s_src=webnav&s_subsrc=datasync'
+  donation_url: 'https://secure3.convio.net/cfrr/site/Donation2?df_id=4620&4620.donation=form1&s_src=webnav&s_subsrc=datasync',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.